go/ephemeral/ephemeral_test.go

```diff
@@ -16,15 +16,44 @@ func TestNewDeviceEK(t *testing.T) {
 	_, err := kbtest.CreateAndSignupFakeUser("t", tc.G)
 	require.NoError(t, err)
 
-	metadata, err := PublishNewDeviceEK(context.Background(), tc.G)
+	publishedMetadata, err := PublishNewDeviceEK(context.Background(), tc.G)
 	require.NoError(t, err)
 
 	fetchedDevices, err := GetOwnDeviceEKs(context.Background(), tc.G)
 	require.NoError(t, err)
 
 	require.Equal(t, 1, len(fetchedDevices))
-	require.Equal(t, metadata, fetchedDevices[0])
-	require.Equal(t, 1, metadata.Generation)
+	for _, fetchedDevice := range fetchedDevices {
```

Comment:
can we setup a second device and publish a key from that test that `GetOwnDeviceEKs` returns both results?